vm/vm.go

```diff
@@ -254,6 +254,9 @@ func (vm *VM) Initialize(
 	ctx, span := vm.tracer.Start(ctx, "VM.Initialize")
 	defer span.End()
 
+	// Set defaults
+	vm.mempool = mempool.New[*chain.Transaction](vm.tracer, vm.config.MempoolSize, vm.config.MempoolSponsorSize)
```

Comment:
would it make sense to have the chain package export a series of type aliases to hide the generic instantiation ?